src/bin/set_referrer.rs

```diff
@@ -0,0 +1,30 @@
+/*
+This is an example of setting a refferal code for a wallet.
```

Comment:
typo `refferal` to `referral` or just remove comment as the functionality is self-evident

Reply:
Done :saluting_face: